ads/strossle.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```

Comment:
2019. Here and everywhere else.

Reply:
@rsimha Got it thanks! Date was fixed.
@lannka Could you please help us here with review and merging? And sorry for the rush, it's just important case.